.k8s/live/cron_jobs/archive_stale.yaml

```diff
@@ -21,6 +21,12 @@ spec:
           - name: cronjob-worker
             image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/laa-get-paid/cccd:app-latest
             imagePullPolicy: Always
+            securityContext:
+              capabilities:
+                drop:
+                - ALL
```

Comment:
It's a good point and maybe it would be neater indented like that, but it's still valid in its current form. We have similar formatting in all these kubernetes config files, where the line following a colon is indented at the same level. Eg on line 20:

```
containers:
- name: cronjob-worker
```